qa/suites/fs/upgrade/mds_upgrade_sequence/branch/quincy.yaml

```diff
@@ -0,0 +1,2 @@
+teuthology:
+  branch: quincy
```

Comment:
What will this get upgraded to? If its to the main branch, then the upgrade going to fail, isn't it?

Reply:
no, it will get to reef, there is lua code added

Reply:
What do you mean? It upgrades the mds keeping the client at the start version.

Reply:
if you check upgrade yamls there is not dest branch mentioned

Reply:
so, its the sha of the branch build which would be the main branch, right?

Reply:
as i mentioned no new changes apart from n-1 and n-2 yamls have been added, quincy always existed in this suite. if it was main then the recent runs should've failed at some point